packages/server/test/unit/config_spec.js

```diff
@@ -1122,6 +1122,7 @@ describe('lib/config', () => {
             viewportHeight: { value: 660, from: 'default' },
             fileServerFolder: { value: '', from: 'default' },
             firefoxGcInterval: { value: { openMode: null, runMode: 1 }, from: 'default' },
+            quiet: { value: false, from: 'default' },
```

Comment:
don't need `quiet` to be a config variable

Reply:
Done